epilogue_1_how_to_get_there_from_here.asciidoc

```diff
@@ -579,12 +579,19 @@ How should use cases interact across a larger system? Is it a problem for one to
     aggregates. When your use case has finished, it can raise an event, and a
     handler elsewhere can run.
 
-Is it a smell for a use case to use multiple repositories, and why?::
-    Yes! A repository is a pattern that we use for reading _aggregates_ from our
-    persistent store. By definition, we should only ever be updating one aggregate
-    at a time. If you need to read data to figure out what to do, then consider a
-    read-model that returns just the data you need, even if you cheat and build it
-    with a repo and domain objects under the hood.
+Is it a smell for a use case to use multiple repositories/aggregates, and why?::
+    An aggregate is a consistency boundary, so if your use case needs to update
+    two aggregates atomically (within the same transaction), then your
+    consistency boundary is wrong.
```

Comment:
```suggestion
   consistency boundary is wrong.  That might be fine ("practicality beats purity") but on the other hand
   it may make your system harder to reason about.
```